build.gradle

```diff
@@ -1,6 +1,13 @@
+import org.gradle.nativeplatform.platform.internal.DefaultNativePlatform
+
+buildscript {
+    ext {
+        ijGradlePluginVersion = (DefaultNativePlatform.currentOperatingSystem.isMacOsX() ? "1.7.0" : "1.12.0")
```

Comment:
build on macos fails with intellij-gradle-plugin > 1.7 with the following error (see https://github.com/redhat-developer/intellij-tekton/pull/645#issuecomment-1405259362):
```
ERROR - oncurrency.BoundedTaskExecutor - Resource not found: /idea/nullApplicationInfo.xml
```